launchable/utils/session.py

```diff
@@ -18,6 +19,25 @@ def _session_file_path(build_name: str) -> Path:
 
 
 def _get_session_id() -> str:
+    # CircleCI changes unix session id each steps, so set non change variable
+    # https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables
+    if os.environ.get("CIRCLECI") is not None:
+        id = os.environ.get("CIRCLE_WORKFLOW_ID")
+        if id is not None:
+             return id
+        Logger().warning("CIRCLECI environment variable is set but not CIRCLE_WORKFLOW_ID")
```

Comment:
If this branch is taken there's no need to do WMI nor Unix session, so better to have this at the top rather than the bottom